sdk/cosmosdb/Azure.ResourceManager.CosmosDB/src/Generated/Models/CosmosDBAccountData.Serialization.cs

```diff
@@ -552,7 +552,7 @@ internal static CosmosDBAccountData DeserializeCosmosDBAccountData(JsonElement e
                         }
                         if (property0.NameEquals("keyVaultKeyUri"u8))
                         {
-                            if (property0.Value.ValueKind == JsonValueKind.Null)
+                            if (property0.Value.ValueKind == JsonValueKind.Null || property0.Value.ValueKind == JsonValueKind.String && property0.Value.GetString().Length == 0)
```

Comment:
Is this something that could be wrapped into a utility method?

Reply:
That would require change in autorest.csharp, it is like this everywhere else.